OpenSim/Common/TimeSeriesTable.h

```diff
@@ -141,6 +141,30 @@ class TimeSeriesTable_ : public DataTable_<double, ETY> {
     TimeSeriesTable_& operator=(const TimeSeriesTable_&) = default;
     TimeSeriesTable_& operator=(TimeSeriesTable_&&)      = default;
     ~TimeSeriesTable_()                                  = default;
+
+    /** Convenience Constructor to efficiently populate a time series table
+    from available data. This is primarily useful for constructing with large
+    data read in from file without having to reallocate and copy memory.*/
+    TimeSeriesTable_(const std::vector<double>& indVec,
+        const SimTK::Matrix_<ETY>& depData,
+        const std::vector<std::string>& labels) : 
+            DataTable_<double, ETY>(indVec, depData, labels) {
+        try {
+            // Perform the validation of the data a TimeSeriesTable
+            this->validateDependentsMetaData();
```

Comment:
I see you added a comment in the base class about validation. Could you add a little more detail, either in the doxygen description for this constructor or for the one in the base class, explaining the situation with calling virtual functions from constructors?